spec/lib/portable_bridge_notation/shared_example_spec.rb

```diff
@@ -0,0 +1,31 @@
+# For now, place to catch conception of full portable bridge notation
+# observer protocol
+
+# guarantee: each method here will be called at most once per
```

Comment:
Oops, this guarantee isn't actually true, even for the example of with_dealt_card.  What's the best way to communicate as part of the protocol the promises regarding order and number of calls?